templates/aspnet-core/files/Controllers/HomeController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using PlatformshAspNetCore.Data;
+using Microsoft.AspNetCore.Mvc;
+using PlatformshAspNetCore.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Caching.Redis;
+
+namespace PlatformshAspNetCore.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly MyDbContext _myDbContext;
```

Comment:
Is underscored property names actually a C# convention?  Most languages these days view it as an anti-pattern, AFAIK.  Let's verify.  (If that's what C# recommends, then that's what we do.)

Reply:
At least for private properties that does seem to be the case https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties#expression-body-definitions

Reply:
Silly C#...